docusaurus/docs/iOS/02-tutorials/02-audio-room-uikit.mdx

```diff
@@ -633,14 +650,18 @@ final class MicButtonView: UIButton {
     @available(*, unavailable)
     required init?(coder: NSCoder) { fatalError("init(coder:) has not been implemented") }
 
-    init() {
+    init(microphone: MicrophoneManager) {
         super.init(frame: .zero)
-        setImage(.init(systemName: "mic.circle"), for: .normal)
+        setImage(.init(systemName: microphone.status == .enabled ? "mic.circle" : "mic.slash.circle"), for: .normal)
```

Comment:
We shouldn't change this one as it's being handled in the next iteration of the microphone manager